tests/RobotFramework/tests/tedge_flows/tedge_flows.robot

```diff
@@ -116,31 +116,38 @@ Consuming messages from a process stdout, periodically
 
 Consuming messages from the tail of file
     # Assuming the flow tail-named-pipe.toml has been properly installed
+    Install Flow    tail-named-pipe.toml
     ${start}    Get Unix Timestamp
     Execute Command    echo hello>/tmp/events
     Should Have MQTT Messages    topic=log/events    message_contains=hello    minimum=1    date_from=${start}
+    [Teardown]    Uninstall Flow    tail-named-pipe.toml
 
 Consuming messages from a file, periodically
     # Assuming the flow read-file-periodically.toml has been properly installed
+    Install Flow    read-file-periodically.toml
     Execute Command    echo hello >/tmp/file.input
     Execute Command    tedge mqtt sub test/file/input --duration 1s | grep hello
     Execute Command    echo world >/tmp/file.input
     Execute Command    tedge mqtt sub test/file/input --duration 1s | grep world
     Execute Command    rm /tmp/file.input
     Execute Command
-    ...    tedge mqtt sub test/file/input --duration 1s | grep 'Error in /etc/tedge/flows/read-file-periodically.toml'
+    ...    tedge mqtt sub test/file/input --duration 1s | grep 'Fail to poll /tmp/file.input'
     Execute Command    echo 'hello world' >/tmp/file.input
     Execute Command    tedge mqtt sub test/file/input --duration 1s | grep 'hello world'
+    [Teardown]    Uninstall Flow    read-file-periodically.toml
 
 Appending messages to a file
     # Assuming the flow append-to-file.toml has been properly installed
+    Install Flow    append-to-file.toml
     Execute Command    for i in $(seq 3); do tedge mqtt pub seq/events "$i"; done
```

Comment:
This might still be flacky, even if okay after 10 runs.

Reply:
> We need to wait for the flow to be fully loaded before sending MQTT messages to the flow input topic.

Aah okay. I thought the flakiness was because we're doing the grep on the output file immediately after publishing the message, without giving enough time for the mapper to process that input and write to the file.

> For that we need to improve tedge flow so status or stats messages are published over MQTT when a flow is loaded.

Yeah, that'd fix both.